app/src/main/java/com/codesoom/assignment/HttpRequest.java

```diff
@@ -0,0 +1,55 @@
+package com.codesoom.assignment;
+
+public class HttpRequest {
+    private final String method;
+    private final String path;
+
+    public HttpRequest(String method, String path) {
+        this.method = method;
+        this.path = path;
+    }
+
+    public boolean isGetAllTasks(){
+        return HttpMethod.GET.getMethod().equals(method) && isTasksPath();
+    }
+
+    public boolean isDeleteTask() {
+        return HttpMethod.DELETE.getMethod().equals(method) && isTasksPathWithId();
+    }
+
+    public boolean isUpdateTask() {
+        return HttpMethod.PUT.getMethod().equals(method) || HttpMethod.PATCH.equals(method) && isTasksPathWithId();
+    }
+
+    public boolean isCreateTask() {
+        return HttpMethod.POST.getMethod().equals(method) && isTasksPath();
+    }
+
+    public boolean isGetOneTask() {
+        return HttpMethod.GET.getMethod().equals(method) && isTasksPathWithId();
+    }
+
+    private boolean isTasksPath() {
+        if("/tasks".equals(path)){
+            return true;
+        }
+        return false;
+    }
```

Comment:
이 메소드는 그냥 이렇게 해도 됩니다.
```suggestion
    private boolean isTasksPath() {
        return "/tasks".equals(path);
    }
```